style.css

```
@@ -15,23 +15,332 @@
+    padding-top: 30px;
+    gap: 30px;
+    max-height: 220px;
+}
+main .article2 div {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    gap: 15px;
+    padding-left: 10px;
+    padding-top: 10px;
+    max-width: 50vw;
+    font-size: 10px;
+    font-family: Roboto, sans-serif;
+    font-size-adjust: 0.58;
+}
+main .article2 span img {
+    max-width: 50vw;
+    max-height: 200px;
+    box-shadow: 2px 10px 10px rgba(0, 0, 0, 0.5);
+}
+.anchor2 {
+    text-decoration: none;
+    color: var(--paars);
+    z-index: 1;
+}
+main .article2::before {
+    display: flex;
+    margin-top: -50px;
+    content: "work";
+    color: var(--grijs);
+    opacity: 18%;
+    font-weight: bold;
+    font-size: 250px;
+    font-family: Roboto, sans-serif;
+    position: absolute;
+}
+/*--------main--article3------------*/
+main .article3 {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+    padding-top: 30px;
+    gap: 30px;
+}
+main .article3 div {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    gap: 15px;
+    padding-left: 10px;
+    padding-top: 10px;
+    max-width: 50vw;
+    font-size: 10px;
+    font-family: Roboto, sans-serif;
+    font-size-adjust: 0.58;
+}
+main .article3 span img {
+    max-width: 50vw;
+    max-height: 200px;
+    box-shadow: 2px 10px 10px rgba(0, 0, 0, 0.5);
+}
+.anchor3 {
+    text-decoration: none;
+    color: var(--paars);
+    z-index: 1;
+    cursor: pointer;
+}
+main .article3::before {
+    display: flex;
+    margin-top: -50px;
+    content: "Variety";
+    color: var(--grijs);
+    opacity: 18%;
+    font-weight: bold;
+    font-size: 250px;
+    font-family: Roboto, sans-serif;
+    position: absolute;
+}
+/*-------------main--article4----------------*/
+main .article4 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding-top: 30px;
+    padding-bottom: 50px;
+    gap: 30px;
+}
+main .article4 div {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    gap: 15px;
+    padding-left: 10px;
+    padding-top: 10px;
+    max-width: 50vw;
+    font-size: 10px;
+    font-family: Roboto, sans-serif;
+    font-size-adjust: 0.58;
+}
+main .article4 span img {
+    max-width: 50vw;
+    max-height: 200px;
+    box-shadow: 2px 10px 10px rgba(0, 0, 0, 0.5);
+}
+.anchor4 {
+    text-decoration: none;
+    color: var(--paars);
+    z-index: 1;
+    cursor: pointer;
+}
+main .article4::before {
+    display: flex;
+    margin-top: -50px;
+    content: "worth";
+    color: var(--grijs);
+    opacity: 18%;
+    font-weight: bold;
+    font-size: 250px;
+    font-family: Roboto, sans-serif;
+    position: absolute;
+}
+/*--------------container-------*/
+.container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 50px 50px 50px 50px;
```

Dit kun je korter schrijven
```suggestion
    padding: 50px;
```